### DISCUSSION TITLE
how to subscript a module

### DISCUSSION THREAD

**Discussion by jgriesfeller** [metno/pyaerocom#672] [help wanted]
Hi, 

for my aeroval parallelization effort I would appreciate a little help. As long as this is not solved we all have to stick to the var name CFG for the aeroval config.

The very preliminary parallelization interface is here:
https://github.com/metno/pyaerocom/blob/griesie_fix_663/pyaerocom/tools/aeroval_parallelise.py

To do some the parallelization I have to load the aeroval config file as a module done here:
https://github.com/metno/pyaerocom/blob/d902f21e5ebfea7cef847550117fcb580f496d35/pyaerocom/tools/aeroval_parallelise.py#L66

Unfortunately modules are not subscript-able, so something like 
```python
 CFG = foo["CFG"]
``` 
does not work. In my case that means that the aeroval configuration has to be in a variable with a fixed name (CFG atm). I find that rather inflexible and would like to make that configurable.
I know that this can in principle be done, but I want to focus on the parallelization for now. So if one of you has a solution to the problem, this will be well appreciated.

**Comment by avaldebe** [comment]
> Unfortunately modules are not subscript-able, so something like
> 
> ```python
>  CFG = foo["CFG"]
> ```
> does not work. [...]

try with the [getattr](https://docs.python.org/3/library/functions.html#getattr) built-in function
``` python
CFG = getattr(foo, "CFG")
```

**Comment by jgriesfeller** [comment]
Works, thanks for the suggestion
